### ISSUE TITLE
Add basic keymaps to Importer

### ISSUE THREAD

**Issue by kasecato** [kasecato/vscode-intellij-idea-keybindings#195]
See https://github.com/kasecato/vscode-intellij-idea-keybindings/issues/120#issuecomment-766296058

**Comment by ijoseph** [comment]
Thanks for your diligence on this :)

**Comment by ijoseph** [comment]
From the other thread

>  some basic key mappings like cut and paste were missing.

but, I *do* see cut and paste in my `Emacs copy.xml` file: 
```xml
<keymap version="1" name="Emacs copy" parent="Emacs">
  <action id="$Copy">
    <keyboard-shortcut first-keystroke="alt c" />
    <keyboard-shortcut first-keystroke="escape" second-keystroke="w" />
    <keyboard-shortcut first-keystroke="meta c" />
  </action>
  <action id="$Cut">
    <keyboard-shortcut first-keystroke="shift delete" />
    <keyboard-shortcut first-keystroke="meta x" />
  </action>
  <action id="$Paste">
    <keyboard-shortcut first-keystroke="alt p" />
    <keyboard-shortcut first-keystroke="ctrl y" />
    <keyboard-shortcut first-keystroke="meta v" />
  </action>
  <action id="$Redo">
    <keyboard-shortcut first-keystroke="shift ctrl z" />
    <keyboard-shortcut first-keystroke="shift alt minus" />
    <keyboard-shortcut first-keystroke="shift meta z" />
  </action>
...
```

perhaps you were speaking generally, though. 

> the default emacs bindings will not be reflected because of it not exported in xml. 

Hm, I couldn't find any that were not reflected in the XML, upon looking. 

```xml
  <action id="EditorNextWord">
    <keyboard-shortcut first-keystroke="alt f" />
    <keyboard-shortcut first-keystroke="escape" second-keystroke="ctrl f" />
    <keyboard-shortcut first-keystroke="alt right" />
  </action>
  <action id="EditorNextWordWithSelection">
    <keyboard-shortcut first-keystroke="shift ctrl right" />
    <keyboard-shortcut first-keystroke="shift alt right" />
  </action>
```

[here](https://file.io/5CcF6u94nuxj) is another copy (of just the XML file this time).

**Comment by kasecato** [comment]
That's true. I added `EditorNextWord`, `EditorNextWordWithSelection` and more. Thank you. 
https://github.com/kasecato/vscode-intellij-idea-keybindings/pull/196/files

**Comment by ijoseph** [comment]
Oh, I see, you meant missing from your mapping file `ActionIdCommandMapping.json`. Gotcha. Ok, I'll try again!

**Comment by kasecato** [comment]
@ijoseph I just released [v1.0.1](https://marketplace.visualstudio.com/items/k--kato.intellij-idea-keybindings/changelog). Thank you for your help. Please check it.
